app/views/categories/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @category.name %></h1>
+<% if @category.short_wiki %>
+  <%= raw(sanitize(QuestionsController.renderer.render(@category.short_wiki), scrubber: scrubber)) %>
+<% end %>
+
+<% post_count = @posts.count %>
+<p class="has-color-tertiary-500" title="<%= post_count %>">
+  <%= number_to_human post_count, precision: 1, significant: false, locale: 'en-ABBREV', units: 'number.human', format: '%n%u' %>
```

Comment:
Something is wrong here, as 12 posts are shown as 1.2 posts:

![Image showing the bug mentioned above](https://user-images.githubusercontent.com/21335202/79335895-15876880-7f23-11ea-8e34-992992538215.png)

Reply:
Seems to be fixed.